handlers.go

```diff
@@ -218,13 +221,33 @@ func setupGatewayHandler(cfg Config, nd *Node, tracingAuth string) (http.Handler
 			if request.Header.Get("Tracestate") != "" {
 				request.Header.Del("Tracestate")
 			}
+			if request.Header.Get(NoBlockcacheHeader) != "" {
+				request.Header.Del(NoBlockcacheHeader)
+			}
+		}
+
+		// Process cache skipping header
+		if noBlockCache := request.Header.Get(NoBlockcacheHeader); noBlockCache != "" {
```

Comment:
Thoughts on the text that should go here? Should it just be `true` or `false`?

Reply:
Perhaps enable on explicit `true`, disable on everything else?
```suggestion
		if noBlockCache := request.Header.Get(NoBlockcacheHeader); noBlockCache == "true" {
```